doajtest/unit/test_regexes.py

```diff
@@ -0,0 +1,34 @@
+""" Gather and test DOAJ regexes here """
+
+from doajtest.helpers import DoajTestCase
+from portality.view.forms import DOI_REGEX
+from doajtest.fixtures import dois
+
+import re
+
+
+class TestRegexes(DoajTestCase):
+
+    def setUp(self):
+        super(TestRegexes, self).setUp()
```

Comment:
These can just be `pass` I think - do you need the index set up and torn down? (Adds 2s per new test.)